src/pages/unseoJang/Login/Login.js

```diff
@@ -2,14 +2,82 @@ import './Login.scss';
 import React, { Component } from 'react';
 
 class Login extends Component {
+  //login handle
+  constructor() {
+    super();
+    this.state = {
+      //state 말 그대로 컴포넌트의 상태를 수정
+      idInputValue: '',
+      pwInputValue: '',
+    };
+  }
+  handleIdInput = e => {
+    //IdInout은 event의 약자 e를 인자로 받음
+    this.setState({
+      idInputValue: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pwInputValue: e.target.value,
+    });
+  };
```

Comment:
input tag에 name property를 부여할 수 있는 것, + 객체 계산된 속성명을 이용해 input handler 지금 id와 pw 두개로 나눠져있는것 하나로 합칠 수 있습니다! 시도해보세요!

Reply:
```suggestion
  handleIdInput = e => {
    //IdInout은 event의 약자 e를 인자로 받음
    this.setState({
      idInputValue: e.target.value,
    });
  };

  handlePwInput = e => {
    this.setState({
      pwInputValue: e.target.value,
    });
  };
```

해당부분 처리하였습니다.
이렇게 하는게 맞는건가 싶네요 ㅎㅎ
확인부탁드리겠습니다.